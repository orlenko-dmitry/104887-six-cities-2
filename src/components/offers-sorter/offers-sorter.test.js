import React from 'react';
import {shallow} from 'enzyme';

import OffersSorter from '../offers-sorter/offers-sorter.jsx';

it(`OffesSorter renders correctly`, () => {
  const tree = shallow(
      <OffersSorter
        isOpen={false}
        onOpenSorterClick={() => {}}
      />
  );

  expect(tree).toMatchSnapshot();
});
