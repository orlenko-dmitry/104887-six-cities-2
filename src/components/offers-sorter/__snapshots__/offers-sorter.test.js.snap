// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OffesSorter renders correctly 1`] = `
<form
  action="#"
  className="places__sorting"
  method="get"
>
  <span
    className="places__sorting-caption"
  >
    Sort by 
  </span>
  <span
    className="places__sorting-type"
    tabIndex="0"
  >
    Popular
    <svg
      className="places__sorting-arrow"
      data-testid="offers-sorter-arrow"
      height="4"
      onClick={[Function]}
      style={Object {}}
      width="7"
    >
      <use
        xlinkHref="#icon-arrow-select"
      />
    </svg>
  </span>
  <ul
    className="places__options places__options--custom "
    data-testid="offers-sorter-list"
  >
    <li
      className="places__option places__option--active"
      tabIndex="0"
    >
      Popular
    </li>
    <li
      className="places__option"
      tabIndex="0"
    >
      Price: low to high
    </li>
    <li
      className="places__option"
      tabIndex="0"
    >
      Price: high to low
    </li>
    <li
      className="places__option"
      tabIndex="0"
    >
      Top rated first
    </li>
  </ul>
</form>
`;
